src/components/ContentHighlights/HighlightStepper/HighlightStepperTitle.jsx

```diff
@@ -13,15 +13,30 @@ const HighlightStepperTitle = () => (
       <Col xs={12} md={8} lg={6}>
         <h3 className="mb-3 d-flex align-items-center">
           <Icon src={EditCircle} className="mr-2 text-brand" />
-          {STEPPER_STEP_TEXT.HEADER_TEXT.createTitle}
+          <FormattedMessage
+            id="highlights.new.highlights.stepper.stepper.step.header.text.create.title"
+            defaultMessage="Create a title for your highlight"
+            description="Create title header message shown to administrators during creation of new content highlights"
+          />
         </h3>
         <div className="mb-4.5">
           <p>
-            {STEPPER_STEP_TEXT.SUB_TEXT.createTitle}
+            <FormattedMessage
+              id="highlights.new.highlights.stepper.stepper.step.sub.text.create.title"
+              defaultMessage="Create a unique title for your highlight. This title is visible
+              to your learners and helps them navigate to relevant content."
+              description="Create title sub message shown to administrators during creation of new content highlights"
+            />
           </p>
           <p>
             <strong>
-              {STEPPER_STEP_TEXT.PRO_TIP_TEXT.createTitle}
+              <FormattedMessage
+                id="highlights.new.highlights.stepper.stepper.step.pro.tip.text.create.title"
+                defaultMessage='Pro tip: we recommend naming your highlight collection to reflect skills
+                it aims to develop, or to draw the attention of specific groups it targets.
+                For example, "Recommended for Marketing" or "Develop Leadership Skills".'
```

Comment:
As our strings are translated through AI. So we are not getting the right translations for quotes and apostrophe. So for that kind of strings we are using variables like.

```js
values={{
              apostrophe: "'",
              doubleQoute: '"',
            }}
```